src/v2-cloud/backend/cf-hono-server.ts

```
@@ -0,0 +1,455 @@
+    let db: SQLDatabase;
+    let cfBackendKey: string;
+    switch (cfBackendMode) {
+      case "DURABLE_OBJECT":
+        {
+          cfBackendKey = c.env.CF_BACKEND_KEY ?? "FP_BACKEND_DO";
+          // console.log("DO-CF_BACKEND_KEY", cfBackendKey, c.env[cfBackendKey]);
+          db = new CFDObjSQLDatabase(getBackendDurableObject(c.env));
+        }
+        break;
+
+      case "D1":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)